src/droid/keepalive.c

```diff
@@ -146,7 +146,7 @@ static void pending_req_reply_cb(DBusPendingCall *pending, void *userdata) {
                                        DBUS_TYPE_INT32, &period,
                                        DBUS_TYPE_INVALID));
 
-    k->timeout = PA_USEC_PER_SEC * (period - 5);
```

Comment:
Could just use the given period. MCE reports N second renew period, but waits N+15 seconds i.e. there is some scheduling slack already added in by default.